### DISCUSSION TITLE
How to remove blank line for sublist?

### DISCUSSION THREAD

**Discussion by hyeokhyen** [greenelab/lab-website-template#223] [Q&A] [answered]
Hi,

I am making a list with a sublist. For example,

- Undergraduate-level linear algebra
- Undergraduate-level multivariate calculus
- Undergraduate-level statistics and probability theory
- Coding experience:
  - Python (required)
  - Matlab (optional)
  - C/C++ (optional)
  - R (optional)

When rendering there is a blank line spacing between 'Coding experience:' and 'python (required)'

Could you help me with this?

**Comment by vincerubinetti** [comment, marked as answer]
This is a CSS "bug". I've noted it in https://github.com/greenelab/lab-website-template/issues/209, but here is a fix you can apply right away. In `_styles/list.scss`, **within** `li { ... }` and under `line-height`, add the following:

```css
ul,
ol {
  margin: 0;
}
```

Normally, lists need a little bit of space (margin) above and below, but this says, when a list is inside another list item, set the space around it back to 0.

**Reply by hyeokhyen** [reply]
Great! I did this:

li {
  margin: 5px 0;
  padding-left: 10px;
  text-align: justify;
  line-height: var(--spacing);
  ul,
  ol {
    margin: 0;
  }
}

And it removed the space :)

Thank you for your quick reply!

**Reply by hyeokhyen** [reply]
By the way, could you also teach me how to remove line space for the new list as well?

**Reply by hyeokhyen** [reply]
I have another question!

I have this script:

Each team has four deliverables on Canvas:
  - a proposal (15%)
  - a short “madness”presentation (10%)
  - a final presentation (25%)
  - a final report. (50%)
<br><br>
Topic: TBD

The 'Topic' is also indented as if it is part of the list. Is there a way to remove the indent for a new paragraph?

**Reply by vincerubinetti** [reply]
> By the way, could you also teach me how to remove line space for the new list as well?

I don't know what you mean by this.

> The 'Topic' is also indented as if it is part of the list. Is there a way to remove the indent for a new paragraph?

Please provide a link to your repo and a screenshot of the problem. For your original problem, by chance I was able to figure out what was going on just from text, but in most cases I need much more detail to be able to help you.
